packages/common/src/service.js

```diff
@@ -51,14 +51,13 @@ function Service(config, options) {
   });
 
   this.makeAuthenticatedRequest = util.makeAuthenticatedRequestFactory(reqCfg);
-
   this.authClient = this.makeAuthenticatedRequest.authClient;
   this.baseUrl = config.baseUrl;
   this.getCredentials = this.makeAuthenticatedRequest.getCredentials;
   this.globalInterceptors = arrify(options.interceptors_);
   this.interceptors = [];
   this.packageJson = config.packageJson;
-  this.projectId = options.projectId;
+  this.projectId = options.projectId || '{{projectId}}';
```

Comment:
That sounds like maybe more effort than we should give this? My only real critique is that it looks like there might be some unnecessary duplication and I think it would be nice if we could consolidate this logic somehow.